### PR TITLE
adds impeachment polls dataset to fivethirtyeight package, 3rd attempt

### PR DESCRIPTION
**Pull request by fatimak98**
Hey @rudeboybert! 

Thanks for your feedback - sorry for all the excess files, not sure why that happened. I followed the steps you left in my last PR attempt. Hopefully this PR works, fingers crossed! Tagging @beanumber

### CODE REVIEW

File: R/data_fatima.R [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#'   \item{end}{Poll end date, the last date responses were collected}
+#'   \item{pollster}{entity/organization that created poll, collected and published data}
+#'   \item{sponsor}{sponsor of pollster}
+#'   \item{sample_size}{number of respondents for each}
+#'   \item{pop}{categorical variable with 3 categories: a, rv, lv -- value unknown}
+#'   \item{tracking}{true/false logical -- value unknown}
+#'   \item{text}{poll question}
+#'   \item{category}{category of poll question with 5 categories: impeach and remove, begin proceedings, begin inquiry, reasons, impeach}
+#'   \item{include}{yes/no logical -- value unknown}
+#'   \item{yes}{% of repsondents in sample who answered "Yes" to the poll question}
```
Comment (rudeboybert) [inline R/data_fatima.R]: Unfortunately it seems all the following `%` signs are causing errors. Could you replace them all with the word `Percent`?

**Review by rudeboybert — COMMENTED** [review]
Hey Fatima, you're almost done! One more change is needed. After making this change, don't forget to build and check your package before committing; this tests that the package is suitable for submission to CRAN.

**Comment by beanumber** [comment]
@fatimak98 please attend to these changes so we can close the book on this. Thanks!

File: R/data_fatima.R (hunk 1 above)
Comment (fatimak98) [reply R/data_fatima.R]: Hi Albert! Sorry for the delay on this, I made the requested change above. Let me know if there are any other changes I need to make.

Comment (rudeboybert) [reply R/data_fatima.R]: Eep! It seems a merge conflict has creeped in as evidenced by the 
```
<<<<<<< HEAD
```
and the
```
=======
```
and the
```
>>>>>>> 25e8defc63e3a707dcecaa074932ec91ea22f474
```
Could you remove these lines, and remove the redundant entries? Almost there!

Comment (fatimak98) [reply R/data_fatima.R]: I've removed the lines and redundant entries (hopefully), please let me know if you can see these changes on your end?

**Review by rudeboybert — APPROVED** [review]
Looks good!

**Comment by rudeboybert** [comment]
Great! You're done! You're now listed as a package [contributor](https://github.com/rudeboybert/fivethirtyeight/graphs/contributors). Thanks for your work @fatimak98!

**Comment by fatimak98** [comment]
Thank you so much @rudeboybert !

**Pull request merged**